__sample__/index.html

```diff
@@ -2,9 +2,12 @@
 <html>
   <head>
     <link rel="stylesheet" href="styles.css">
-    <title>sample code</title>
+    <title>prep-example</title>
   </head>
   <body>
+    <h1>I would like to build</h1>
+    <p>an app that showcases another marketplace for college student housing. This would create an easier way for landlords and
+    tenants to advertise and search what they need.</p>
```

Comment:
Hi Adrienne, looks like your customization made it into the sample HTML file instead of prep-example/index.html